inc/classes/class_func.php

```diff
@@ -396,7 +449,6 @@ public function text2html($string, $mode = 0)
             }
 
             if ($mode != 2) {
-                $img_start = "<img src=\"design/".$auth["design"]."/images/";
```

Comment:
Why was this removed?

Reply:
Because the variable `$img_start` is not in use at all.
In this line, it gets written, but no one (in the complete file) is accessing/reading this variable.
See https://github.com/lansuite/lansuite/blob/master/inc/classes/class_func.php#L399

Reply:
Ah, understood!